packages/core/src/constants/blockExplorers.ts

```diff
@@ -46,6 +47,10 @@ export const etherscanBlockExplorers: Record<EtherscanChains, BlockExplorer> = {
     name: 'Etherscan',
     url: 'https://kovan-optimistic.etherscan.io',
   },
+  optimismGoerli: {
+    name: 'Etherscan',
+    url: 'https://l2-explorer.surge.sh/',
```

Comment:
This isn't an official Etherscan block explorer so let's remove this from here. Can inline this URL on the `blockExplorers.default` field for now.

Reply:
Should I just use 'https://goerli-optimistic.etherscan.io' in the `url` field here instead? Or pass an empty string?

Also, what do you mean by inlining in `blockExplorers.default`? Like inline comment?

Reply:
Etherscan doesn't have a block explorer out yet for Goerli Optimism so no need to add to the Etherscan block explorer object.